<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaidInfo35</class>
 <widget class="QWidget" name="RaidInfo35">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxDay4_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Day 4 (1st)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="labelIVCountDay4_1">
        <property name="text">
         <string>IV Count</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelAbilityDay4_1">
        <property name="text">
         <string>Ability</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelCharacteristicDay4_1">
        <property name="text">
         <string>Characteristic</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelRaidDay4_1">
        <property name="text">
         <string>Raid</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinBoxHPDay4_1">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAtkDay4_1">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxDefDay4_1">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpADay4_1">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpDDay4_1">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpeDay4_1">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBoxCharacteristicDay4_1"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelNatureDay4_1">
        <property name="text">
         <string>Nature</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxRaidDay4_1"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxAbilityDay4_1"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxIVCountDay4_1">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxNatureDay4_1"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelCheck">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxDay4_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Day 4 (2nd)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinBoxHPDay4_2">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAtkDay4_2">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxDefDay4_2">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpADay4_2">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpDDay4_2">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpeDay4_2">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRaidDay4_2">
        <property name="text">
         <string>Raid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxRaidDay4_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelIVCountDay4_2">
        <property name="text">
         <string>IV Count</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxIVCountDay4_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelNatureDay4_2">
        <property name="text">
         <string>Nature</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxNatureDay4_2"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelAbilityDay4_2">
        <property name="text">
         <string>Ability</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxAbilityDay4_2"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelCharacteristicDay4_2">
        <property name="text">
         <string>Characteristic</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxCharacteristicDay4_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBoxDay5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Day 5</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="spinBoxHPDay5">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAtkDay5">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxDefDay5">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpADay5">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpDDay5">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpeDay5">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRaidDay5">
        <property name="text">
         <string>Raid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxRaidDay5"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelIVCountDay5">
        <property name="text">
         <string>IV Count</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxIVCountDay5">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelNatureDay5">
        <property name="text">
         <string>Nature</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxNatureDay5"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelAbilityDay5">
        <property name="text">
         <string>Ability</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxAbilityDay5"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelCharacteristicDay5">
        <property name="text">
         <string>Characteristic</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxCharacteristicDay5"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="groupBoxDay6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Day 6</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="spinBoxHPDay6">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAtkDay6">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxDefDay6">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpADay6">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpDDay6">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxSpeDay6">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRaidDay6">
        <property name="text">
         <string>Raid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxRaidDay6"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelIVCountDay6">
        <property name="text">
         <string>IV Count</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxIVCountDay6">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelNatureDay6">
        <property name="text">
         <string>Nature</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxNatureDay6"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelAbilityDay6">
        <property name="text">
         <string>Ability</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxAbilityDay6"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelCharacteristicDay6">
        <property name="text">
         <string>Characteristic</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxCharacteristicDay6"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxHPDay4_1</tabstop>
  <tabstop>spinBoxAtkDay4_1</tabstop>
  <tabstop>spinBoxDefDay4_1</tabstop>
  <tabstop>spinBoxSpADay4_1</tabstop>
  <tabstop>spinBoxSpDDay4_1</tabstop>
  <tabstop>spinBoxSpeDay4_1</tabstop>
  <tabstop>comboBoxRaidDay4_1</tabstop>
  <tabstop>spinBoxIVCountDay4_1</tabstop>
  <tabstop>comboBoxNatureDay4_1</tabstop>
  <tabstop>comboBoxAbilityDay4_1</tabstop>
  <tabstop>comboBoxCharacteristicDay4_1</tabstop>
  <tabstop>spinBoxHPDay4_2</tabstop>
  <tabstop>spinBoxAtkDay4_2</tabstop>
  <tabstop>spinBoxDefDay4_2</tabstop>
  <tabstop>spinBoxSpADay4_2</tabstop>
  <tabstop>spinBoxSpDDay4_2</tabstop>
  <tabstop>spinBoxSpeDay4_2</tabstop>
  <tabstop>comboBoxRaidDay4_2</tabstop>
  <tabstop>spinBoxIVCountDay4_2</tabstop>
  <tabstop>comboBoxNatureDay4_2</tabstop>
  <tabstop>comboBoxAbilityDay4_2</tabstop>
  <tabstop>comboBoxCharacteristicDay4_2</tabstop>
  <tabstop>spinBoxHPDay5</tabstop>
  <tabstop>spinBoxAtkDay5</tabstop>
  <tabstop>spinBoxDefDay5</tabstop>
  <tabstop>spinBoxSpADay5</tabstop>
  <tabstop>spinBoxSpDDay5</tabstop>
  <tabstop>spinBoxSpeDay5</tabstop>
  <tabstop>comboBoxRaidDay5</tabstop>
  <tabstop>spinBoxIVCountDay5</tabstop>
  <tabstop>comboBoxNatureDay5</tabstop>
  <tabstop>comboBoxAbilityDay5</tabstop>
  <tabstop>comboBoxCharacteristicDay5</tabstop>
  <tabstop>spinBoxHPDay6</tabstop>
  <tabstop>spinBoxAtkDay6</tabstop>
  <tabstop>spinBoxDefDay6</tabstop>
  <tabstop>spinBoxSpADay6</tabstop>
  <tabstop>spinBoxSpDDay6</tabstop>
  <tabstop>spinBoxSpeDay6</tabstop>
  <tabstop>comboBoxRaidDay6</tabstop>
  <tabstop>spinBoxIVCountDay6</tabstop>
  <tabstop>comboBoxNatureDay6</tabstop>
  <tabstop>comboBoxAbilityDay6</tabstop>
  <tabstop>comboBoxCharacteristicDay6</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
